src/chord/node.cljs

```diff
@@ -0,0 +1,42 @@
+(ns chord.node
+  (:require [cljs.nodejs :as nodejs]
+            [chord.channels :as channels]
+            [chord.format :as format]))
+
+(defn ws-server [ws-opts {:keys [read-ch write-ch :as chord-opts]} handler]
+  (when-let [ws (nodejs/require "ws")]
+    (let [server (ws.Server. (clj->js ws-opts))]
+      (.on server "connection"
+           (fn [socket request]
+             (let [ws-ch (channels/wrap-websocket socket chord-opts #(.close socket))]
+               (handler ws-ch request))))
+      server)))
+
+(comment
+  "Plain example"
+
+  (defn handler [ws-ch req]
```

Comment:
These are great examples - could you also put them in the README to make them more discoverable?